src/main/kotlin/io/hsar/wh40k/combatsimulator/random/RandomDice.kt

```diff
@@ -43,5 +44,20 @@ object RandomDice {
      */
     fun roll(diceString: String): Int = parser.parse(diceString, roller).totalRoll
 
+    fun randomBodyPart(): BodyPart {
```

Comment:
I completely forgot about this - I think this method isn't in the right place though, it should probably be calculated before (ideally as per the book by reversing the number) and only stored here.

Side note: this might need to change to be a sealed class rather than an enum.

Reply:
Hmmmm - I like the current implementation of using roll results to abstract away the raw number rolled. I agree that there are better places perhaps for the function and I've lifted it into AttackExecutor, but I can't see the harm in just doing another roll for it - it's equally random and allows us to use a RollResult for the hit roll rather than a raw Int